CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_shopify_holistic_reporting v0.6.0
+[PR #21](https://github.com/fivetran/dbt_shopify_holistic_reporting/pull/21) includes the following changes:
+
+## Dependency Updates
+Updates the underlying Shopify package version range from [">=0.10.0", "<0.11.0"] to [">=0.11.0", "<0.14.0"] to account for breaking changes introduced to the Shopify package up to v0.14.0. This wider range will while accommodate previous versions while supporting an upcoming release to Shopify, which will not have breaking changes for this package.
```

Comment:
The updates to v0.13.0 of Shopify while being a minor update will technically not be breaking. Therefore, just want to clarify that and add details as to why we are making the exception here to provide a wider range.

Additionally, I have included my suggestion for the Shopify Source range callout as a sub-bullet. 
```suggestion
- Updates the underlying Shopify package version range from [">=0.10.0", "<0.11.0"] to [">=0.11.0", "<0.14.0"] to account for changes introduced to the Shopify package up to v0.14.0. This wider range will accommodate previous versions while supporting an upcoming release to Shopify, which will not have breaking changes for this package.
   - Additionally, please note that the wider range for the Shopify dbt package also updates the underlying Shopify Source dependency range from versions [">=0.10.0", "<0.11.0"] to  [">=0.11.0", "<0.13.0"].
```

Reply:
updated!